README.md

```diff
@@ -630,92 +630,92 @@ This report is autogenerated and includes tokenizers and detokenizers tests. The
     <tr>
       <td >Tiktoken</td>
       <td >Qwen/Qwen-14B-Chat</td>
-      <td >98.17</td>
-      <td >109</td>
+      <td >92.91</td>
+      <td >141</td>
     </tr>
     <tr>
       <td >Tiktoken</td>
       <td >Salesforce/xgen-7b-8k-base</td>
-      <td >98.17</td>
-      <td >109</td>
+      <td >95.20</td>
+      <td >125</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >ProsusAI/finbert</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >91.43</td>
+      <td >105</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >bert-base-multilingual-cased</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >91.43</td>
+      <td >105</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >bert-base-uncased</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >91.43</td>
+      <td >105</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >cointegrated/rubert-tiny2</td>
-      <td >91.36</td>
-      <td >81</td>
+      <td >91.43</td>
+      <td >105</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >distilbert-base-uncased-finetuned-sst-2-english</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >91.43</td>
+      <td >105</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >google/electra-base-discriminator</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >91.43</td>
+      <td >105</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >google/mobilebert-uncased</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >94.38</td>
+      <td >89</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >jhgan/ko-sbert-sts</td>
-      <td >87.65</td>
-      <td >81</td>
+      <td >91.43</td>
+      <td >105</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >prajjwal1/bert-mini</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >94.38</td>
+      <td >89</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >rajiv003/ernie-finetuned-qqp</td>
-      <td >97.53</td>
-      <td >81</td>
+      <td >94.38</td>
+      <td >89</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >rasa/LaBSE</td>
-      <td >90.12</td>
-      <td >81</td>
+      <td >80.00</td>
```

Comment:
is it OK that pass rate is decreased?

Reply:
There is no real degradation for this tokenizer. For some reason, numpy sometimes lets elementwise comparison for differently shaped tensors. There was one test (from `test_hf_wordpiece_tokenizers_multiple_strings`) that successfully passed `np.all((ov_result := ov_tokenized[output_name]) == hf_result)` for `(6, 1)` and `(6, 0)` shaped tensors, resulting in 5/8 pass rate (should be 4/8). I added a shape check and new test cases, so in this PR the pass rate is 16/32. That is why the total pass rate is lower.